style/americana.js

```diff
@@ -133,7 +159,8 @@ var style = {
   bearing: 0,
   sources: {
     openmaptiles: {
-      url: "https://api.maptiler.com/tiles/v3/tiles.json?key=" + mapTilerKey,
+      url: "http://localhost:8080/data/v3.json",
+      //      url: "https://api.maptiler.com/tiles/v3/tiles.json?key=" + mapTilerKey,
```

Comment:
Consider: https://www.npmjs.com/package/dotenv

So that you can put the localhost URL in your working directory `.env` file (and `.gitignore` it), and the deployment process can use the production URL.